sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/executable/SonarTSCoreBundle.java

```diff
@@ -93,6 +97,14 @@ public SonarTSRunnerCommand getSonarTsRunnerCommand(String tsconfigPath, Iterabl
     return runnerCommand;
   }
 
+  private String getNodeExecutable() {
+    String nodeExecutable = settings.getString(TypeScriptPlugin.NODE_EXECUTABLE);
+    if (nodeExecutable == null) {
```

Comment:
@saberduck do we need this logic? SQ should provide default value

Reply:
Indeed. Removed.